mizar/common/common.py

```diff
@@ -382,3 +383,11 @@ def reset_param(param):
     param.extra = None
     param.return_message = None
     return param
+
+def network_policy_rpc_helper(conf, conf_list):
```

Comment:
suggest to name the function more specifically; its purpose seems to check whether it would exceed the max length __IF__ one more element appends. How about name it like "conf_list_has_max_elements"?